docs/user-guide/mvd-configuration.md

```diff
@@ -38,16 +20,12 @@ If the mediation layer is not used, ZSS directly at `https://<zowe.externalDomai
 
 ### app-server configuration
 
-The app-server uses the Zowe server configuration file for customizing server behavior. For a full list of parameters, requirements, and descriptions, see [the json-schema document for the app-server](https://github.com/zowe/zlux/blob/v2.x/staging/schemas/zlux-config-schema.json) which describes attributes that can be specified within the configuration file section `components.app-server`
```

Comment:
Multiple links are not found with these changes. Please see the broken links report.